configure.ac

```diff
@@ -18,8 +18,11 @@ PKG_CHECK_MODULES([lzma], [liblzma])
 PKG_CHECK_MODULES([zlib], [zlib])
 AC_ARG_ENABLE(
 	[bzip2],
-	AS_HELP_STRING([--disable-bzip2],[Do not use bzip2 compression (uses bzip2 by default)])
+	AS_HELP_STRING([--disable-bzip2],[Do not use bzip2 compression (uses bzip2 by default)]),
+	BZIP='no',
+	BZIP='yes'
```

Comment:
The semantics of AC_ARG_ENABLE are confusing, so these simple assignments won't work as expected. The action in the third argument is run if either "--enable-bzip2" or "--disable-bzip2" is passed. Or, if neither option is passed, the action in the fourth argument is run.

I would recommend not setting either third or fourth arguments (the current state), and checking the "enable_bzip2" variable instead, which is automatically defined. See the AS_IF macro use below for an example.